charts/scalar-admin-for-kubernetes/templates/_helpers.tpl

```diff
@@ -0,0 +1,116 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "scalar-admin-for-kubernetes.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
```

Comment:
Ditto. Because it is automatically generated by the Helm, I want to keep this sentence as is.

You can see the default sentence in the following way.

```console
$ sed -n 11p ./foo/templates/_helpers.tpl
If release name contains chart name it will be used as a full name.
```